build/linux/arch/PKGBUILD

```diff
@@ -0,0 +1,53 @@
+# Maintainer: Meriadec Pillet <meriadec.pillet@gmail.com>
+# shellcheck disable=SC2154,SC2034,SC2164
+
+pkgname=ledger-live
+pkgver=1.0.3
+pkgrel=1
+pkgdesc="Open source companion app for your Ledger devices"
+arch=('x86_64')
+url="https://www.ledgerwallet.com/live"
+license=('MIT')
+makedepends=(yarn python2)
+
+# TODO generate changelog from release notes
+changelog=
+
+source=("https://github.com/LedgerHQ/ledger-live-desktop/archive/v${pkgver}.tar.gz"
+        "ledger-live.desktop")
+md5sums=('51ff80d0db79e82c52c1732b52ff1174'
+         '52705147909a0a988907a23a71199092')
+# TODO sign with ledger pgp
+validpgpkeys=()
```

Comment:
Didn't find any source about signing github archives, so if you want to sign it we will probably have to host the archive ourself...